animation_nodes/nodes/mesh/insert_custom_attribute.py

```diff
@@ -89,6 +96,16 @@ def execute(self, mesh, customAttributeName, data):
 
         if self.dataType == "INT":
             _data = VirtualLongList.create(data, 0).materialize(amount)
+        elif self.dataType == "INT32_2D":
+            if self.useDataList:
+                indices1 = LongList.fromValues(data.asNumpyArray()[::2])
+                indices2 = LongList.fromValues(data.asNumpyArray()[1::2])
+            else:
+                indices1 = LongList.fromValues([data[0]])
+                indices2 = LongList.fromValues([data[1]])
+            _indices1 = VirtualLongList.create(indices1, 0)
+            _indices2 = VirtualLongList.create(indices2, 0)
+            _data = createEdgeIndices(amount, _indices1, _indices2)
```

Comment:
This seems a bit too complicated. Can we just create a virtual list for int2?
See the `virtual_clist_types.json` file. And maybe add Int2 to `math/vector.pxd` to make things easier.